pkg/controller/template/render_test.go

```diff
@@ -99,6 +99,10 @@ func TestCloudProvider(t *testing.T) {
 		platform:          configv1.NonePlatformType,
 		featureGateAccess: externalDisabledFG,
 		res:               "",
+	}, {
+		platform:          configv1.ExternalPlatformType,
+		featureGateAccess: externalEnabledFG,
```

Comment:
I think you need to add `cloudprovider.ExternalCloudProviderFeatureExternal` to `externalEnabledFG`/externalDisabledFG

Reply:
@adriengentil there is a strange behavior when adding this case, the result of [renderTemplate](https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/template/render_test.go#L136-L143) is empty, even if adding it to `externalEnabledFG` and `externalDisabledFG`. Am I missing something?